app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/elements/element-dialog/descriptions/FormattedDate.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+import React from "react";
+import { useTranslation } from "@i18n/client";
+import { ExampleWrapper } from "./ExampleWrapper";
+import {
+  FormattedDate as FormattedDateComponent,
+  Description,
+  Label,
+} from "@clientComponents/forms";
+
+export const FormattedDate = () => {
+  const { t } = useTranslation("form-builder");
+  return (
+    <>
+      <h3 className="mb-0">{t("addElementDialog.formattedDate.title")}</h3>
+      <p>{t("addElementDialog.formattedDate.description")}</p>
+
+      <ExampleWrapper className="mt-4">
+        <Label htmlFor="formattedDate">{t("addElementDialog.formattedDate.label")}</Label>
+        <Description>{t("addElementDialog.formattedDate.description")}</Description>
```

Comment:
```suggestion
```

Reply:
Form element description, not hint text :) 

<img width="518" alt="Screenshot 2024-08-22 at 3 13 18 PM" src="https://github.com/user-attachments/assets/25f583e1-1af7-4ec9-a784-f8b4c18ff17c">